app/javascript/courses/review/components/CoursesReview__ChecklistEditor.res

```diff
@@ -151,147 +186,208 @@ let make = (~reviewChecklist, ~updateReviewChecklistCB, ~closeEditModeCB, ~targe
+                      />
+                    </div>
                   </div>
-                  <button
-                    title={t("checklist_title.remove_button_title")}
-                    className="bg-gray-200 p-2 w-11 border border-gray-400 text-gray-700 rounded ml-2 hover:text-red-600 hover:bg-red-100 focus:outline-none"
-                    onClick={_ => removeChecklistItem(itemIndex, setState)}>
-                    <i className="fas fa-trash-alt" />
-                  </button>
-                </div>
-                <div>
-                  {ReviewChecklistItem.result(reviewChecklistItem)
-                  ->Js.Array2.mapi((resultItem, resultIndex) => {
-                    let feedback = Belt.Option.getWithDefault(
-                      ReviewChecklistResult.feedback(resultItem),
-                      "",
-                    )
-                    <Spread
-                      props={"data-result-item": string_of_int(resultIndex)}
-                      key={string_of_int(itemIndex) ++ string_of_int(resultIndex)}>
-                      <div className="pl-2 md:pl-4 mt-2">
-                        <div className="flex">
-                          <label
-                            title={t("disabled")}
-                            className="flex-shrink-0 rounded border border-gray-400 bg-gray-100 w-4 h-4 mr-2 mt-3 cursor-not-allowed"
-                          />
-                          <div className="w-full bg-gray-100 relative">
-                            <div className="relative">
-                              <input
-                                className="checklist-editor__checklist-result-item-title h-10 pr-12 focus:outline-none focus:bg-white focus:border-primary-300"
+                  <div>
+                    {ReviewChecklistItem.result(reviewChecklistItem)
+                    ->Js.Array2.mapi((resultItem, resultIndex) => {
+                      let feedback = Belt.Option.getWithDefault(
+                        ReviewChecklistResult.feedback(resultItem),
+                        "",
+                      )
+                      <Spread
+                        props={"data-result-item": string_of_int(resultIndex)}
+                        key={string_of_int(itemIndex) ++ string_of_int(resultIndex)}>
+                        <div className="pl-2 md:pl-4 mt-2">
+                          <div className="flex">
+                            <label
+                              title={t("disabled")}
+                              className="flex-shrink-0 rounded border border-gray-400 bg-gray-100 w-4 h-4 mr-2 mt-3 cursor-not-allowed"
+                            />
+                            <div className="w-full bg-gray-100 relative">
+                              <div className="relative">
+                                <input
+                                  className="checklist-editor__checklist-result-item-title h-10 pr-12 focus:outline-none focus:bg-white focus:border-primary-300"
```

Comment:
```suggestion
                                  className="checklist-editor__checklist-result-item-title h-10 pr-20 focus:outline-none focus:bg-white focus:border-primary-300"
```

Reply:
Updated layout of `checklist-result-item-title` with flexbox, so padding hack isn't needed anymore.